user_data/events.csv

```diff
@@ -0,0 +1,53 @@
+subject,session_datetime,trial_id,event_id,event_start_time,event_type
```

Comment:
Could you please remove `event_id`?  I believe we opted to not use `event_id` and just have `event_start_time`.